### PR TITLE
To do interactive list

### PR DESCRIPTION
**Pull request by djo1975**
Description
In this project, you will add some functionality to your application to make it interactive. The user will also be able to mark task completion by selecting the corresponding checkbox (or undo it by unchecking the checkbox). The updated tasks list will be stored in local storage.

IMPORTANT NOTE: Read all requirements before you start building your project.

General requirements
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
HTML/CSS requirements
Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
JavaScript requirements
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).
Add event listener to the checkbox (change).
Update items object's value for completed key upon user actions.
Implement a function for the "Clear all completed" button (use filter() method).
Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).

### CODE REVIEW

File: src/status.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-cycle
+import { tasks } from './index.js';
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (jerryowusu) [inline src/status.js]: As part of the general requirements for this and all other projects, kindly do not disable the linters. The linters are there to guide you in order to produce the best standard practices. Kindly consider removing the lines 1 and 4 and work within the limits of the linters. :pray:

File: src/index.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
 import './style.css';
+// eslint-disable-next-line import/no-cycle
+import { toggleTaskStatus } from './status.js';
 
 const taskList = document.querySelector('.container2');
 
 // Initialize tasks array
-let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+// eslint-disable-next-line import/prefer-default-export, import/no-mutable-exports
+export let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
```
Comment (jerryowusu) [inline src/index.js]: As part of the general requirements for this and all other projects, kindly do not disable the linters. The linters are there to guide you in order to produce the best standard practices. Kindly consider removing the lines 2 and 8 and work within the limits of the linters. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @djo1975 , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

CookedDelectableDolphin-size_restricted


## Highlights
All linters are passing :heavy_check_mark: 
Project working as expected :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @djo1975 , 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**